Core.Auditing/IfxAuditLogger.cs

```diff
@@ -116,5 +118,26 @@ public static string FetchIPAddress()
 
             return ipAddress;
         }
+        public void LogRequest(ITelemetryClient telemetryClient, OperationResult operationResult, TargetResource[] targetResources, CallerIdentity[] callerIdentities, AuditMandatoryProperties auditMandatoryProperties, AuditOptionalProperties auditOptionalProperties = null)
+        {
+            string webAppName = Environment.GetEnvironmentVariable("WEBSITE_SITE_NAME");
+            if (string.IsNullOrEmpty(webAppName))
+            {
+                telemetryClient?.LogWarning($"[{nameof(LogRequest)}] Web app name isn't found from environment variable.");
```

Comment:
If telemetry client should never be null then we do not need the nullable operator here.